rafx-framework/Cargo.toml

```diff
@@ -31,6 +31,9 @@ serde_bytes = "0.11"
 glam = { version = "0.13.1", features = [ "serde" ] }
 parking_lot = "0.11.1"
 slotmap = "1.0.2"
+downcast-rs = "1.2.0"
+dashmap = "4.0.2"
+rustc-hash = "1.1.0"
```

Comment:
I think we should try to use FNV consistently everywhere (or something else consistently everywhere) unless there is extremely good reason to add more of them.

Reply:
In my testing, `rustc-hash` has been significantly faster than `FNV`, e.g.
```
// 5.3909 milliseconds to query the world for 150000 entities using 1 parallel `query_visibility`. (std::hashmap)
// ~4.5 milliseconds to query the world for 150000 entities using 1 parallel `query_visibility`. (fnvhashmap)
// 2.3005 milliseconds to query the world for 150000 entities using 1 parallel `query_visibility`. (identity hash)
// 2 milliseconds to query the world for 150000 entities using 1 parallel `query_visibility`. (fxhashmap)
```
My vote is therefore with `rustc-hash`.

Reply:
I'm happy to switch if the quality is still ok, I just think we should use FNV in this change and switch everything over as a separate change. I also think we should typedef something in rafx-base and change everything to use that, rather than changing all code to use a different crate directly.

Reply:
Replaced with `fnv` for now.

Reply:
https://github.com/aclysma/rafx/issues/138